fabric-entity-events-v1/src/main/java/net/fabricmc/fabric/mixin/entity/event/LivingEntityMixin.java

```diff
@@ -86,7 +86,7 @@ private void onIsSleepingInBed(CallbackInfoReturnable<Boolean> info) {
 		BlockPos sleepingPos = getSleepingPosition().orElse(null);
 
 		if (sleepingPos != null) {
-			BlockState bedState = world.getBlockState(sleepingPos);
+			BlockState bedState = ((LivingEntity) (Object) this).world.getBlockState(sleepingPos);
```

Comment:
I don't think mixin ever guaranteed remapability for shadows from supermixins; the mixin should preferably extend the superclass of the target class, and such references should be remapped properly.

Reply:
This change was not supposed to be in this PR, i was on the wrong branch.